docs/reference/known-issues/known-issues.md

```diff
@@ -0,0 +1,9 @@
+(reference-known-issues)=
+# Known Issues
+
+## gRPC Version
+
```

Comment:
In this section, can you remove the extra line breaks? I know it renders the same on the docs site, but since our docs repo can have community contributions, it's important to keep this stuff consistent across the whole docs. We just use standard paragraph formatting